pkg/controllers/vrpq/query_reconciler.go

```diff
@@ -33,6 +33,11 @@ import (
 	ctrl "sigs.k8s.io/controller-runtime"
 )
 
+const (
+	querying     = "Querying"
+	successQuery = "Query successful"
```

Comment:
Suggest including state to denote that they are used in the state status field.
```suggestion
	stateQuerying     = "Querying"
	stateSuccessQuery = "Query successful"
```